src/plugins/geomap/_base.scss

```diff
@@ -142,6 +142,7 @@
 
 .olControlAttribution {
 	bottom: 5px;
+	font-size: x-small;
```

Comment:
Ok, if it's consistent with the plugin itself then I can merge as it but it should be made consistent with the rest of the plugins for next version. Sounds good?